client/constants/index.js

```diff
@@ -1,6 +1,12 @@
 export const confirmApiUrl = "/api/v1/{orgSlug}/account/password/reset/confirm";
+export const genericError = "Error occurred!";
 export const loginApiUrl = "/api/v1/{orgSlug}/account/token";
+export const loginError = "Login error occurred.";
+export const loginSuccess = "Login success.";
+export const logoutSuccess = "Logout.";
 export const passwordConfirmError = "The two password fields didn't match.";
 export const registerApiUrl = "/api/v1/{orgSlug}/account/";
+export const registerError = "Registration error.";
+export const registerSuccess = "Registration success";
```

Comment:
Is it correct to store error and success messages here (passwordConfirmError was here initially) or we should store them in `org-configuration`, so we can add localization, or maybe somewhere else?